File: Bird.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjectClassAnimals
{
    class Bird:Animal
    {
        public Bird()
        {
            Console.WriteLine("A bird was created");
        }

        public override void Eats()
        {
            Console.WriteLine("Our birds eats worms and seeds");
        }

        public override void Sleep()
        {
            Console.WriteLine("A bird sleeps in a nest or on a branch");
        }

        public override void Sound()
        {
            Console.WriteLine("Cip Cip Cip");
        }

        public override void Trick()
        {
            Console.WriteLine("Roll Roll and Back");
        }
    }
}

File: Zoo/src/Bird.java

public class Bird implements Animal{
	private String name = "Bird";
	
	public String sound() {
        return "Cip-Cip";
    }
    
    public String eats() {
        return "cereals";
    }
    
    public String sleep() {
        return "in the tree";
    }

    @Override
    public String getName() {
        return name;
    }


}
